src/openscad.cc

```diff
@@ -280,29 +280,29 @@ int cmdline(const char *deps_output_file, const std::string &filename, const cha
 
 	ExportFileFormatOptions exportFileFormatOptions;
 	FileFormat curFormat;
-	std::string extsn;
+	std::string formatName;
 	std::string output_file_str = output_file;
 	const char *new_output_file = nullptr;
 
 	if(!export_format.empty()) {
-		PRINT("Using extension of --export-format option");
-		extsn = export_format;
+		PRINTB("Using format '%s' of --export-format option", export_format.c_str());
```

Comment:
As this is simply a confirmation that OpenSCAD understood the `--export-format` option, I wonder if we could simply remove it

Reply:
I guess that's fine too, as unknown values are warned in https://github.com/openscad/openscad/blob/d21e65b4ce7aa8cc33731feabc60e8a38d819eab/src/openscad.cc#L1018 already.

Reply:
Indeed, it's perhaps best to just remove this message.